### PR TITLE
This Week in Information Security - Week of June 1st

### PR DESCRIPTION
**Pull request by cneill**
__Bio:__
Charles Neill is a Security Engineer at Rackspace, who tests Rackspace products as well as OpenStack projects like Barbican. He is also a developer on [DefectDojo](https://github.com/rackerlabs/django-DefectDojo), an open source defect tracking system using the Django framework.

__Social Media:__
Twitter: [@ccneill](https://twitter.com/ccneill)
Github: [cneill](https://github.com/cneill)
Website: [inventropy.us/blog/](https://inventropy.us/blog/)

__Editing Preference:__
Feel free to edit as needed.

__Posting Preference:__
Desired date to publish: June 2nd, 2015

I tried adding some YouTube videos to the mix this week, so that there's more than just text to entice potential readers. Let me know if I've inadvertently gotten the syntax wrong, etc. (I made sure to include my "more" tag this week :blush:  ) Also, I realize the blurbs are pretty long this week for some of the articles I listed, so if you'd like me to trim them down at all, just let me know.

Thanks!

### CODE REVIEW

File: src/site_source/_posts/2015-06-01-this-week-in-information-security.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+As always, you can find me on Twitter [@ccneill][twitter] if you have any thoughts on this post!
+
+<!-- more -->
+
+# News / Opinions
+
+- [__Section 215 expires - for now__][nsa] - EFF is celebrating the lapse of section 215 of the USA PATRIOT act, which authorized the NSA's phone metadata collection program, though they acknowledge that the USA FREEDOM Act will likely reinstate some of these powers in different ways and with more oversight. A vote on the USA FREEDOM Act will likely happen sometime this week. This comes on the heels of a recent [federal appeals court decision][nsa_court] stating that the metadata program was not authorized by federal law, and went beyond the bounds of what legislators thought they were voting for in authorizing the USA PATRIOT act.
```
Comment (catlook) [inline src/site_source/_posts/2015-06-01-this-week-in-information-security.md]: ...  law and went beyond ...

File: src/site_source/_posts/2015-06-01-this-week-in-information-security.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+- [__Users of Hola VPN service being used for DDoS attack__][hola] - Hola is a VPN service with millions of users that works in a similar way to [Tor][tor], in that users of the service become endpoints that traffic can pass through anonymously. Unlike Tor, the user's participation as a router of anonymized traffic is not an opt-in system, and the user may be unaware that all kinds of terrible traffic might be passing through their connection. This post on 8chan __(Warning: other pages on the 8chan site may be unsafe for work)__ describes a system set up by the Hola creators that allows anyone to pay to send traffic through their anonymous network, which boasts over 9 million unique IPs. This service was apparently used to send massive amounts of traffic to 8chan, ultimately crashing the site. The traffic could not be easily correlated to Hola endpoints, because it bears no extra headers, and endpoints lack common open ports like Tor, forcing the 8chan site to require the solving of a CAPTCHA to create new posts.
```
Comment (catlook) [inline src/site_source/_posts/2015-06-01-this-week-in-information-security.md]: ...no extra headers. Furthermore, endpoints lack ...

File: src/site_source/_posts/2015-06-01-this-week-in-information-security.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+- [__The creator of Locker ransomware renonunces his creation__][locker] - The author of the ransomware known as [Locker][locker_2] has apologized for releasing it, and has provided the Bitcoin addresses it uses, public and private keys, and details about how files are encrypted, claiming that all currently encrypted files will be decrypted starting on June 2nd at midnight. __Warning:__ I have not downloaded the file referenced in this pastebin, and take no responsibility if it is indeed malicious. Obviously you should always be cautious when downloading files online, but it's especially true if you already know the author is a prolific malware creator.
```
Comment (catlook) [inline src/site_source/_posts/2015-06-01-this-week-in-information-security.md]: ... releasing it and has provided the following information: the Bitcoin addresses it uses, public and private keys, and details about how files are encrypted.  The author also claims that all currently ...
.... Obviously, you should always ...

File: src/site_source/_posts/2015-06-01-this-week-in-information-security.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+- [__Stegosploit__][stegosploit1] - [@therealsaumil][saumil] has released some excellent research at Hack in the Box showing that one can insert browser exploits into a simple image, and have them execute when the user loads said image in their browser. This is not like previous research where an attacker simply dumps PHP or ASP code into EXIF data and then spoofs the file extension, but uses HTML5 Canvas to parse JavaScript that can't be detected with a simple EXIF data search, as he demonstrates in the video below.
```
Comment (catlook) [inline src/site_source/_posts/2015-06-01-this-week-in-information-security.md]: .... but, instead, it uses HTML5 Canvas ...

File: src/site_source/_posts/2015-06-01-this-week-in-information-security.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+{% youtube fAyuOhB4uvo %}
+
+Now to find a [Ghostery][ghostery]-like plugin to block scary images...
+
+- [__Mobile application assessments by the numbers - a whole-istic view__][dan_talk] - I was fortunate enough to attend a San Antonio OWASP talk by DenimGroup CTO Dan Cornell ([@danielcornell][dan_twitter]) last week about his company's experience auditing mobile applications for security issues, and the lessons they've learned along the way. The presentation takes data from over 60 engagements with 20 different mobile applications to try to determine what methods work best for finding the most critical security bugs. Unfortunately, it appears that there is no single approach that will find all the bugs you probably care about - it's a combination of manual and automated testing, and both source code analysis and black-box testing. The main points I took away from the talk were that the majority of serious issues affecting mobile applications actually come from the server side, not the application itself, and that the tooling for mobile applicaton testing is not as mature as that used to test classic web applications, for example.
```
Comment (catlook) [inline src/site_source/_posts/2015-06-01-this-week-in-information-security.md]: ...  issues and the lessons  ...
... testing and both source ...

File: src/site_source/_posts/2015-06-01-this-week-in-information-security.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+[stegosploit1]: https://conference.hitb.org/hitbsecconf2015ams/wp-content/uploads/2015/02/D1T1-Saumil-Shah-Stegosploit-Hacking-with-Pictures.pdf
+[saumil]: https://twitter.com/therealsaumil
+[ghostery]: https://www.ghostery.com/en/
+[dan_talk]: http://www.slideshare.net/denimgroup/application-security-assessments-by-the-numbers-owaspappseceu20151
+[dan_twitter]: https://twitter.com/danielcornell
+
+[php_sqli]: https://paragonie.com/blog/2015/05/preventing-sql-injection-in-php-applications-easy-and-definitive-guide
+[easydb]: https://github.com/paragonie/easydb
+
+[cheetah]: https://www.youtube.com/watch?v=_luhn7TLfWU
```
Comment (catlook) [inline src/site_source/_posts/2015-06-01-this-week-in-information-security.md]: [ Done!!  Carry on :) ]

**Comment by cneill** [comment]
Thanks so much, Cat! I just committed all the suggested edits.

**Comment by rgbkrk** [comment]
I'll let le @smashwilson look over this.

**Comment by smashwilson** [comment]
Pulling into a PR on the deconst-hosted repo presently :wrench:

**Comment by cneill** [comment]
Dang, it looks like I messed up by naming the post the same as the week before, so clicking the "read more" on the old post takes you to the new one (also, the Disqus comment on the new article is from the old one). What would be the best way to prevent name collisions going forward?

**Comment by smashwilson** [comment]
Oh, we'll probably need to do another dual PR to update the title.

I... might be able to write a Jekyll plugin to detect collisions in the future, but I'm swamped for a while, so I'm not sure where that sits on the priority queue...

**Pull request merged**